utils/node_local_test.py

```diff
@@ -416,6 +416,10 @@ def get_base_env(clean=False):
     env['D_LOG_MASK'] = 'DEBUG'
     env['D_LOG_SIZE'] = '5g'
     env['FI_UNIVERSE_SIZE'] = '128'
+
+    # Otherwise max number of contexts will be limited by number of cores
+    env['CRT_CTX_NUM'] = 32
```

Comment:
This is an environment variable and should be a string.
```suggestion
    env['CRT_CTX_NUM'] = '32'
```